pom.xml

```diff
@@ -113,6 +113,9 @@ THE SOFTWARE.
     <access-modifier-checker.version>1.21</access-modifier-checker.version>
     <junit.jupiter.version>5.7.1</junit.jupiter.version>
     <powermock.version>2.0.9</powermock.version>
+
+    <!-- SSHD plugin version -->
+    <sshd.plugin.version>3.0</sshd.plugin.version>
```

Comment:
```suggestion
    <sshd.plugin.version>3.0.1</sshd.plugin.version>
```

?

Reply:
BTW I cannot find the source tree for this update.

Reply:
not published yet as I need an incremental to set the jenkins.version in the plugin I believe?

Reply:
unless I just skip tests and publish with the next weekly?

Reply:
Once https://github.com/jenkinsci/jenkins/runs/1961167481 finishes you should have something you can use.